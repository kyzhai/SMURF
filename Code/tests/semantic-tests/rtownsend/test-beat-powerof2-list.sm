/* Lists of beats and ints are allowed if the ints are all powers of 2 */

x = [2,4,8.]
main = []
