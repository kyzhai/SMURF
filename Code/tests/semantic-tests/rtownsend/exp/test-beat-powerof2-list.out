Program: 
	SVardef: 
		ID: x  :: [Beat]

	[2, 4, 8.]
	SMain: []
Global Scope: 
	ID: main  :: []
	ID: x  :: [Beat]
	ID: print x :: a -> a
	ID: random  :: Int

